pkg/api/mockhttpclient.go

```diff
@@ -98,8 +99,9 @@ func (c *MockHTTPClient) sendMultiple(request Request) (*Response, error) {
 
 	fileName := c.Responses[c.RespCount].FileName
 
+	dat := []byte(c.Responses[c.RespCount].RespData)
```

Comment:
Are you adding an option for a string that you want returned for a call?

Reply:
Yes, however a RespFile setting would still override this